style.css

```diff
@@ -89,4 +89,363 @@ body {
+    line-height: 52px;
+    text-align: left;
+    color: #172b4d;
+  }
+
+  .description-about {
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    color: #344563;
+    margin-top: 10px;
+  }
+
+  .skill-list,
+  .lang {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    gap: 25px;
+    padding-top: 30px;
+  }
+
+  .lang li a {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 20px;
+    background-color: #f7f7f9;
+    border-radius: 5px;
+    text-decoration: none;
+    color: #253858;
+    padding: 5px;
+    font-size: 20px;
+    font-weight: 500;
+  }
+
+  .list {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .list span {
+    color: #000;
+    font-size: 20px;
+    font-weight: 500;
+  }
+
+  hr {
+    background: #dfe1e6;
+    height: 1px;
+    border: 0.5px;
+    margin-top: 10px;
+  }
+
+  /* BUTTON STYLES */
+  .submit-btn:enabled {
+    background: #fff;
+    color: #7f8cff;
+  }
+
+  .submit-btn:disabled {
+    background: #c1c7d0;
+    color: #fff;
+  }
+
+  .submit-btn:hover {
+    background-color: #6070ff;
+    color: #fff;
+  }
+
+  .submit-btn:active {
+    transform: scale(1.1);
+    background: #2230d2;
+    color: #fff;
+    -webkit-transform: scale(1);
+  }
+
+  .btn.btn-default.btn-hover:enabled {
+    background: #fff;
+    color: #7f8cff;
+  }
+
+  .btn.btn-default.btn-hover:disabled {
+    background: #fff;
+    color: #c1c7d0;
+    border: 1px solid #c1c7d0;
+    border-radius: 8px;
+  }
+
+  .btn.btn-default.btn-hover:hover {
+    background-color: #6070ff;
+    color: #fff;
+    border-color: #fff;
+  }
+
+  .btn.btn-default.btn-hover.btn-pressed:active {
+    transform: scale(1.1);
+    background: #2230d2;
+    color: #fff;
+    -webkit-transform: scale(1);
+  }
+
+  .btn.btn-default.btn-hover.btn-pressed:focus {
+    outline: none;
+    box-shadow: none;
+  }
+
+  /* Contact Section */
+  .contact-me {
+    background: url(Assets/images/contact-form-shapes-mobile.svg) #6070ff;
+    background-repeat: no-repeat;
+    border-top-left-radius: 120px;
+    padding: 114px 24px;
+    background-position: 100% 10%;
+    height: max-content;
+  }
+
+  .contact-me h1 {
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    text-align: center;
+    color: #fff;
+  }
+
+  .contact-me p {
+    text-align: center;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    color: #fff;
+    margin-top: 10px;
+  }
+
+  .contact-form {
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    margin-top: 30px;
+  }
+
+  #message {
+    height: 170px;
+  }
+
+  input[type=text],
+  input[type=email],
+  select,
+  textarea {
+    font-family: "Poppins", sans-serif;
+    width: 100%;
+    padding: 12px;
+    box-sizing: border-box;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    border-radius: 10px;
+    border: 1px solid #cfd8dc;
+  }
```

Comment:
- Kindly minimize the space between your input field and remove additional size on bottom to stick the Figma design and have better UI 👍 


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/lucy-sees/Webforlio/assets/52010789/907bb66d-42cc-48e0-86b0-97cf4244d3fe" width="200">|<img src="https://github.com/lucy-sees/Webforlio/assets/52010789/f328740a-ad10-4c20-bdf1-1d6a91e50d8d" width="200">|


😎️

🤝️